### PR TITLE
The message about untrusted domains now has a tip in it.

### PR DESCRIPTION
**Pull request by kacperpaczos**
* Resolves: # 

## Summary
I've added a brief message about the possibility of missing headers (for example, in the case of reverse proxy)

## TODO
Add Another translations?

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ x] Screenshots before/after for front-end changes
image

- [ x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
https://github.com/nextcloud/documentation/pull/10032
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)
Not

### CODE REVIEW

File: core/l10n/en_GB.json
Comment (artonge) [inline core/l10n/en_GB.json:1]: No need to update l10n files, so you can revert those changes

Comment (kacperpaczos) [reply core/l10n/en_GB.json:1]: Done

Comment (artonge) [reply core/l10n/en_GB.json:1]: Looks like `core/l10n/en_GB.json` is still here.

Comment (kacperpaczos) [reply core/l10n/en_GB.json:1]: Please, check now last commit.

Comment (artonge) [reply core/l10n/en_GB.json:1]: I did, and the locales files are deleted entirely. They should not be deleted

Comment (artonge) [reply core/l10n/en_GB.json:1]: Changes in `untrustedDomains.php` are ok.
But please do not change anything under the l10n directory.

Comment (kacperpaczos) [reply core/l10n/en_GB.json:1]: Ok. Did this PR is ok now?

Comment (artonge) [reply core/l10n/en_GB.json:1]: No, two files are deleted under the l10n directory.

Comment (artonge) [reply core/l10n/en_GB.json:1]: In doubt, see the "Files changed" tab in GitHub

**Pull request closed without merging**